### ISSUE TITLE
Q: How do I set OpenTrack for flipped/inverted Clip?

### ISSUE THREAD

**Issue by samkatakouzinos** [opentrack/opentrack#1211]
Question: How to adjust OpenTrack settings so my Delanclip Fusion works well upside down?
Upside down as in the normally bottom longer arm is on top and the normally shorter arm is on bottom? Or flip it 180°in simpler terms?
If I can be pointed in the right direction that would be great. I can't figure it out right now.

**Comment by GO63-samara** [comment]
I would try resizing the clip like so:
![image](https://user-images.githubusercontent.com/63638386/103444009-17928280-4c7e-11eb-9b8f-42413b71c88d.png)

**Comment by samkatakouzinos** [comment]
> I would try resizing the clip like so:

This works as I hoped and very well. 
Thank you very much.
